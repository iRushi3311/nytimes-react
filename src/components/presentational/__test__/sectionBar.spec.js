import React from 'react';
import renderer from 'react-test-renderer';
import SectionBar from '../section-bar.js';
import {MemoryRouter} from 'react-router-dom';

describe('>>>C O M P O N E N T S --- snapshot test for SectionBar', () => {

    let componentTree;

    beforeEach(() => {
        componentTree = renderer.create(
            <MemoryRouter>
                <SectionBar />
            </MemoryRouter>).toJSON();
    });

    it('+++ SectionBar match with snapshot ', () => {
        expect(componentTree).toMatchSnapshot();
    });

});