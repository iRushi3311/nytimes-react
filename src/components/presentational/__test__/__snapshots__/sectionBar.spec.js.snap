// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>C O M P O N E N T S --- snapshot test for SectionBar +++ SectionBar match with snapshot  1`] = `
<ul
  className="sc-bdVaJa TTSOL"
>
  <li
    className="selected-section sc-bwzfXH jxoMPG"
    onClick={[Function]}
  >
    <a
      href="/"
      onClick={[Function]}
    >
       
      Home
       
    </a>
  </li>
  <li
    className="sc-bwzfXH jxoMPG"
    onClick={[Function]}
  >
    <a
      href="/"
      onClick={[Function]}
    >
       
      World
       
    </a>
  </li>
  <li
    className="sc-bwzfXH jxoMPG"
    onClick={[Function]}
  >
    <a
      href="/"
      onClick={[Function]}
    >
       
      U.S.
       
    </a>
  </li>
  <li
    className="sc-bwzfXH jxoMPG"
    onClick={[Function]}
  >
    <a
      href="/"
      onClick={[Function]}
    >
       
      Politics
       
    </a>
  </li>
  <li
    className="sc-bwzfXH jxoMPG"
    onClick={[Function]}
  >
    <a
      href="/"
      onClick={[Function]}
    >
       
      N.Y.
       
    </a>
  </li>
  <li
    className="sc-bwzfXH jxoMPG"
    onClick={[Function]}
  >
    <a
      href="/"
      onClick={[Function]}
    >
       
      More
       
    </a>
  </li>
</ul>
`;
